admin/articles.rb

```diff
@@ -456,6 +456,15 @@ def update
       end
     end
 
+    def destroy
+      article = Goldencobra::Article.find(params[:id])
+      if article.has_children
```

Comment:
Is this necessary? Would it be possible to just rely on the `before_destroy` callback? Maybe add something to the `errors`?

Reply:
Also, this behaviour would be needed for all models that might have children. To keep it dry, maybe use a [`Concern`](http://api.rubyonrails.org/classes/ActiveSupport/Concern.html)?